src/Client/Web/Bit.Client.Web.BlazorUI/Components/Inputs/BitTextField.razor

```diff
@@ -0,0 +1,40 @@
+﻿@namespace Bit.Client.Web.BlazorUI
+@inherits BitComponentBase
+<div @ref="Element"
+     style="@GetElementStyle()"
+     class="@GetElementClass() @FocusClass">
+    @if (IsMultiLine)
+    {
```

Comment:
It's also required to check if (Type == TextFieldType.Text) in order to render a textarea.